src/config/default_planner.yaml

```diff
@@ -49,11 +49,17 @@ motion:
   angle_threshold: 0.01  # How much the robot has to rotate to be considered "rotating"
   min_steps_not_moving: 2  # How many steps the robot has to not move before we consider it "stopped"
   joint_tolerance:
-    arm: 0.05
-    base_x: 0.05
-    lift: 0.05
-    wrist_roll: 0.25
-    wrist_pitch: 0.25
```

Comment:
can you just copy this and make an ok_robot_planner.yaml which is used in your script?